### ISSUE TITLE
Implementations not being picked up

### ISSUE THREAD

**Issue by FredCoen** [primitivefinance/hardhat-marmite#10]
Doesen't find any implementation if docs from README.md with pattern"

` @start:<Greater-than>
        if (newBar > 0) {
            bar = newBar;
        }
        @end`

is used with a ":" 

Two possible fixes, modify regex or disallow ":" after start tag in the docs

PRs
https://github.com/primitivefinance/hardhat-marmite/pull/8
https://github.com/primitivefinance/hardhat-marmite/pull/9

**Comment by clemlak** [comment]
Nice catch! This is actually a typo in the docs, the `:` should not be here.

**Comment by andy-uphold** [comment]
Yup. I just came across this issue too. 8 days and you still haven't removed one character from the README? ;-)
